README.md

```diff
@@ -19,31 +19,67 @@ _Xamarin Forms provides cross-platform APIs, but this project are only worked on
+- [CircleSurfaceView](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.CircleSurfaceView.html) : A View that can have several CircleSurfaceItem, not Page.
+- [CircleToolbarItem](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.CircleToolbarItem.html) : An item class that can show a toolbar item for fits in the circular screen.
+- [ColorMenuItem](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.ColorMenuItem.html) : A item class that allows you to set the background color of the button used in the popup control.
+- [ContentButton](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.ContentButton.html) : A Button that allows you to customize the view to show.
+- [ContextPopupEffectBehavior](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.ContextPopupEffectBehavior.html) : A behavior to show a small popup  has one or two button sticky with any control.
+- [FlatViewCell](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.FlatViewCell.html) : An ViewCell that turns off the default effect(fish-eye effect) applied in wearable profile.
+- [GoogleMapView](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.GoogleMapView.html) : A View to display google javascript map.
+- [InformationPopup](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.InformationPopup.html) : A popup that has a control to show progress and one button on the bottom side of the circular screen.
+- [MediaPlayer](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.MediaPlayer.html) : A class that provides the essential components to play the media contents.
+- [MediaView](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.MediaView.html) : A View to display a video on the screen.
+- [PopupEntry](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.PopupEntry.html) : An Entry that opens popup for editing text of the entry.
+- [TwoButtonPage](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.TwoButtonPage.html) : A Page that has two buttons. One is on the left, and right side of the circular screen.
+- [TwoButtonPopup](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.TwoButtonPopup.html) : A popup that has two buttons. One is on the left, and right side of the circular screen.
+- [Toast](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.Toast.html) : A popup for simple feedback.<br/>
+
+![widgets](doc/design/data/cui_widgets.png)
+
+Tizen Wearable CircularUI project also provides extended controls from Xamarin.Forms.
+These controls looks same with controls povided by Xamarin.Forms, but it provides some additional features for Tize wearable.
```

Comment:
These controls basically look same as what it is in Xamarin.Forms, but provide additional features just for Tizen wearables. The developers would want to use the following controls only when they specifically need to use the provided features in each controls.